.github/workflows/build.yml

```
@@ -34,8 +34,8 @@ jobs:
         env:
           CIBW_PROJECT_REQUIRES_PYTHON: ">=3.7, <3.10"
           CIBW_BUILD: "cp*"
-          CIBW_BEFORE_ALL_MACOS: brew install mysql gcc
-          CIBW_BEFORE_ALL_WINDOWS: choco install mysql-python
+          CIBW_BEFORE_ALL_MACOS: brew install mysql gcc openssl@1.1
+          CIBW_BEFORE_ALL_MACOS: export LIBRARY_PATH=$LIBRARY_PATH:/usr/local/opt/openssl/lib/
```

```suggestion
          CIBW_BEFORE_ALL_MACOS: |
            brew install mysql gcc openssl@1.1
            export LIBRARY_PATH=$LIBRARY_PATH:/usr/local/opt/openssl/lib/
```
Why not use [newer version](https://formulae.brew.sh/formula/openssl@3#default) though? Also I still don't think this is correct. I don't use mac but usually installation command install binaries/libs on /usr/bin and /usr/lib, not /usr/local/opt